documentation/changelog.rst

```diff
@@ -3,6 +3,15 @@
 FlexMeasures Changelog
 **********************
 
+v0.15.1 | August XX, 2023
+============================
+
+Bugfixes
+-----------
+
+* Disable HiGHS logs on the standard output when `LOGGING_LEVEL=INFO` [see `PR #824 <https://github.com/FlexMeasures/flexmeasures/pull/824>`_]
+* Use the `source` to filter beliefs in the `AggregatorReporter` [see `PR #819 <https://github.com/FlexMeasures/flexmeasures/pull/819>`_]
```

Comment:
We had missed applying that filter, but also we fixed the application of the `AggregatorReporter` on cases where input sensors contained data from multiple sources (by applying `one_deterministic_belief_per_event=True`). Actually, the latter fix might be relevant to other reporters, too?

Reply:
Indeed, this could be relevant for the `ProfitReporter` as well.